funcx_endpoint/funcx_endpoint/executors/high_throughput/executor.py

```diff
@@ -471,46 +471,6 @@ def _start_local_interchange_process(self):
             self.address, self.worker_result_port
         )
 
-    def _start_remote_interchange_process(self):
```

Comment:
This appears to be referenced here:
* https://github.com/funcx-faas/funcX/blob/main/funcx_endpoint/tests/integration/test_executor.py#L13
* https://github.com/funcx-faas/funcX/blob/main/funcx_endpoint/tests/integration/test_executor_passthrough.py#L22

We should probably remove those lines (maybe files?) as well.

Reply:
@joshbryan-globus I've removed the basic `test_executor.py` test and updated the `test_executor_passthrough`. I've removed the `_start_remote_interchange_process` call from these as well.